_data/reference.yml

```diff
@@ -49,24 +70,20 @@ toc:
 
 - title: Kubernetes Components
   section:
-  - docs/admin/kube-apiserver.md
-  - docs/admin/kube-controller-manager.md
-  - docs/admin/kube-proxy.md
-  - docs/admin/kube-scheduler.md
-  - docs/admin/kubeadm.md
-  - docs/admin/federation-apiserver.md
-  - docs/admin/federation-controller-manager.md
-  - docs/federation/api-reference/index.md
-  - title: kubelet
+  - title: kubectl CLI Reference
```

Comment:
currently this does:

- Kubernetes components
  - kubectl CLI Reference
     - apiserver
     - kube-proxy
     - ...anything but kubectl

doesn't make sense. Can we just create a title `Command-line tools` and put things in here. We don't talk about kubectl here.